eq-author/src/components/buttons/Button/index.js

```diff
@@ -97,6 +111,7 @@ const Button = styled.button`
   ${props => props.variant === "secondary" && secondaryButton};
   ${props => props.variant === "tertiary" && tertiaryButton};
   ${props => props.variant === "tertiary-light" && tertiaryLightButton};
+  ${props => props.variant === "success" && successButton};
```

Comment:
As much as I like the idea of a "success"! button 😂 is it worth adding a new one here or can we just overwrite the colors of one of the other buttons?